java/src/test/java/ai/rapids/cudf/HostMemoryBufferTest.java

```diff
@@ -169,20 +169,125 @@ public void testFilemap() throws Exception {
 
   @Test
   public void testCopyWithStream() {
+    long length = 1 * 1024 * 1024;
+    long available = PinnedMemoryPool.getAvailableBytes();
+    if (available < (length * 2)) {
+      if (PinnedMemoryPool.isInitialized()) {
+        PinnedMemoryPool.shutdown();
+      }
+      PinnedMemoryPool.initialize(length * 2 + 2048);
+    }
```

Comment:
nit, maybe a utility function for this piece?

Reply:
@revans2 I think this one got missed. Do you think it's worth making the above lines a util function so it can be reused in the tests below?

Reply:
@abellina are you sure, because I created a utility function `initPinnedPoolIfNeeded` for it. Perhaps you just need to refresh.

Reply:
oh! I see it now. Sorry about that.